src/main/cpp/mechanisms/noteManager/decoratormods/noteManager_feederIntake_State.cpp

```diff
@@ -67,7 +67,16 @@ bool noteManagerfeederIntakeState::AtTarget()
 
 bool noteManagerfeederIntakeState::IsTransitionCondition ( bool considerGamepadTransitions )
 {
+	bool transition = false;
 	// To get the current state use m_mechanism->GetCurrentState()
 
-	return ( considerGamepadTransitions && TeleopControl::GetInstance()->IsButtonPressed ( TeleopControlFunctions::EXAMPLE_MECH_FORWARD ) );
+	auto currentstate = m_mechanism->GetCurrentState();
+	bool noSensorsDetected = m_mechanism->launcherSensor->Get() == false && m_mechanism->placerInSensor->Get() == false && m_mechanism->placerMidSensor->Get() == false && m_mechanism->placerOutSensor->Get() == false && m_mechanism->backIntakeSensor->Get() == false && m_mechanism->frontIntakeSensor->Get() == false;
+
+	if((TeleopControl::GetInstance()->IsButtonPressed(TeleopControlFunctions::INTAKE) && m_mechanism->m_scoringMode == RobotStateChanges::ScoringMode::Launcher) || (noSensorsDetected && currentstate == m_genState->GetnoteManager()->STATE_PLACER_TO_LAUNCHER_FRONT))
```

Comment:
Update from main and change
m_mechanism->m_scoringMode == RobotStateChanges::ScoringMode::Launcher
to
m_mechanism->isLauncherMode()